i18n/languages/woocommerce-services.pot

```diff
@@ -4,1124 +4,13 @@ msgid ""
 msgstr ""
 "Project-Id-Version: _s woocommerce-services\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-03-31T20:32:49.044Z\n"
+"POT-Creation-Date: 2020-06-15T16:43:59.511Z\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "PO-Revision-Date: 2014-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 
-#: dist/woocommerce-services-1.23.0.js:107
```

Comment:
@Ferdev do you know why the i18n strings are all removed? Is this supposed to only show up in release zips but somehow they were added to `develop`?

Reply:
I'll investigate this!

Reply:
I think `i18n/` is under `.gitignore`, and our `prerelease` npm script will regenerate them. I think these are okay to be removed in `develop`.

Reply:
Yeah, agree!